resources/js/components/AddJournalModalComponent.vue

```diff
@@ -1,31 +1,77 @@
 <template>
-    <modal-component>
-        <div slot="header">Add new record</div>
-        <div slot="body">  
-            <form id="add-journal-form"> 
-                <datepicker name="date" format="yyyy-MM-dd" placeholder="Date" class="form-control"></datepicker>
-                <input name="database_column" class="form-control mt-2" type="text" placeholder="Text">
-                <textarea name="plan_next_day" class="form-control mt-2" id="exampleFormControlTextarea1" rows="3" placeholder="Plan for next day "></textarea>
-                <input name="goal_tomorrow" class="form-control mt-2" type="text" placeholder="Goal for tomorrow">
-                <div class="form-check mt-2">
-                    <input name="is_achieved" class="form-check-input" type="checkbox" value="" id="defaultCheck1">
-                    <label class="form-check-label" for="defaultCheck1">
-                    Did you achieve today's goal?
-                    </label>
-                </div>
-            </form>
+  <modal-component style="color:black">
```

Comment:
Can you add class here instead of style ? Place the class inside the ModalComponent.